java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Hotfix the modular RPMs release comparison
```

Comment:
a bsc# is not present, is this intended?

Reply:
there is no bsc